### PR TITLE
Make printer width configurable

### LINKED ISSUE #1026: Make printer width configurable
### Feature description

When executing, `dbt run`, I find that the printer width (hardcoded to 80) is too small given the length of my schema and table names. I would like to make this configurable so that the results (RUN, SUCCESS, FAIL, SKIP) align in a column on the right-hand side instead of being staggered.

Here is the relevant line:
https://github.com/fishtown-analytics/dbt/blob/6454a81593e0311ac1d796b43f95a32cb8be4b53/dbt/ui/printer.py#L62

### Example

default width of 80
```
11:02:57 | Concurrency: 4 threads (target='stg_dbt')
11:02:57 | 
11:02:57 | 1 of 3 START table model DBT_SCRATCH_PUBLIC.TOTAL_UNIT_DAILY_TEMP.... [RUN]
11:07:23 | 1 of 3 OK created table model DBT_SCRATCH_PUBLIC.TOTAL_UNIT_DAILY_TEMP [SUCCESS 1 in 265.90s]
11:07:23 | 2 of 3 START table model DBT_SCRATCH_PUBLIC.TOTAL_UNIT_TIME_SERIES_DAILY [RUN]
11:07:58 | 2 of 3 OK created table model DBT_SCRATCH_PUBLIC.TOTAL_UNIT_TIME_SERIES_DAILY [SUCCESS 1 in 34.27s]
11:07:58 | 3 of 3 START table model DBT_SCRATCH_PUBLIC.TOTAL_UNIT_TIME_SERIES_MONTHLY [RUN]
11:08:18 | 3 of 3 OK created table model DBT_SCRATCH_PUBLIC.TOTAL_UNIT_TIME_SERIES_MONTHLY [SUCCESS 1 in 20.77s]
11:08:18 | 
11:08:18 | Finished running 3 table models in 327.56s.
```

hand-updated width of 120
```
11:02:57 | Concurrency: 4 threads (target='stg_dbt')
11:02:57 | 
11:02:57 | 1 of 3 START table model DBT_SCRATCH_PUBLIC.TOTAL_UNIT_DAILY_TEMP............................................ [RUN]
11:07:23 | 1 of 3 OK created table model DBT_SCRATCH_PUBLIC.TOTAL_UNIT_DAILY_TEMP....................................... [SUCCESS 1 in 265.90s]
11:07:23 | 2 of 3 START table model DBT_SCRATCH_PUBLIC.TOTAL_UNIT_TIME_SERIES_DAILY..................................... [RUN]
11:07:58 | 2 of 3 OK created table model DBT_SCRATCH_PUBLIC.TOTAL_UNIT_TIME_SERIES_DAILY................................ [SUCCESS 1 in 34.27s]
11:07:58 | 3 of 3 START table model DBT_SCRATCH_PUBLIC.TOTAL_UNIT_TIME_SERIES_MONTHLY................................... [RUN]
11:08:18 | 3 of 3 OK created table model DBT_SCRATCH_PUBLIC.TOTAL_UNIT_TIME_SERIES_MONTHLY.............................. [SUCCESS 1 in 20.77s]
11:08:18 | 
11:08:18 | Finished running 3 table models in 327.56s.
```

### Who will this benefit?

Anyone else who has long names and wants more readable console output.

### PR DESCRIPTION
**Pull request by kasanchez**
closes #1026

Tests to come. 
I'm open to feedback!

### CODE REVIEW

File: core/dbt/ui/printer.py [hunk 1]
```diff
@@ -13,11 +13,16 @@
...
 COLOR_RESET_ALL = dbt.ui.colors.COLORS['reset_all']
 
+PRINTER_WIDTH = 80
+
 
 def use_colors():
     global USE_COLORS
     USE_COLORS = True
 
+def printer_width(printer_width):
```
Comment (drewbanin) [inline core/dbt/ui/printer.py]: pep8: can you add an extra newline above and below this function?

**Comment by kasanchez** [comment]
hey @drewbanin thanks for the feedback! I could use a pointer on where to add some tests. I'll update my pr to address the pep8 comments above as well

**Comment by drewbanin** [comment]
@kasanchez you can make some tiny additions to this function: https://github.com/fishtown-analytics/dbt/blob/dev/stephen-girard/test/unit/test_config.py#L229

I'd try editing the `default_profile_data['config']` dict to set a custom `printer_width`, then add an `assertEqual` call below to make sure that the value you pass in comes out the other side :)

**Comment by drewbanin** [comment]
@kasanchez what's the happs with this PR? I think we'll want to rebase it against the `dev/wilt-chamberlain` branch -- let me know if you'd like a hand with it!

The code here all LGTM, but looks like something funky happened with the tests :/

**Comment by drewbanin** [comment]
@kasanchez I merged this over in https://github.com/fishtown-analytics/dbt/pull/1459 with a tiny additional test. I did some git magic to make sure you get credit for the commit -- thanks for your contribution to dbt!!

# 🎉

**Pull request closed without merging**